tests/unittests/clients/vertex/test_chat_completions.py

```diff
@@ -0,0 +1,164 @@
+import json
+from typing import Optional
+from unittest.mock import Mock, patch, ANY
+
+import httpx
+import pytest
+
+from ai21 import AI21VertexClient
+from ai21.clients.vertex.ai21_vertex_client import DEFAULT_GCP_REGION, AsyncAI21VertexClient
+from ai21.clients.vertex.gcp_authorization import GCPAuthorization
+from ai21.models.chat import ChatMessage
+from tests.unittests.commons import FAKE_CHAT_COMPLETION_RESPONSE_DICT, FAKE_AUTH_HEADERS
+
+_TEST_REGION = "test-region"
+_TEST_PROJECT = "test-project"
+_FULL_GCP_URL = f"https://{_TEST_REGION}-aiplatform.googleapis.com/v1/projects/{_TEST_PROJECT}/locations/{_TEST_REGION}/publishers/ai21/models/jamba-instruct:rawPredict"
+
+
+def test__vertex_client__when_no_region_provided__should_use_default_region():
+    client = AI21VertexClient()
+    assert client._region == DEFAULT_GCP_REGION
+
+
+@pytest.mark.parametrize(
```

Comment:
I think that parameterization might be redundant here.
If it's only 2 values for 1 test, and we're using this during the test, I'd just write 2 tests.